job_templates/regenerate_puppet_certificate.erb

```diff
@@ -0,0 +1,51 @@
+<%#
+name: Regenerate puppet certificate
+job_category: Miscellaneous
+snippet: false
+template_inputs:
+- name: Delete certificate on Puppet CA
+  required: true
+  input_type: user
+  description: If yes is selected, Foreman will contact Puppet CA of a host and remove
+    the certificate from there. However this requires safe mode to be disabled.
+  options: "no\r\nyes"
```

Comment:
Would be nice if Foreman accepted a list as well and joined it with the correct newlines if needed. The ```\r``` is uncommon in the Unix world.

Reply:
I hope that does not block this PR

Reply:
It doesn't